### PR TITLE
Adding Huddle US

### PR DESCRIPTION
**Pull request by seanspeaks**
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@seanspeaks There are some files of other connectors are added in your PR, Can you please make sure that you have followed the instructions mentioned in the [documentation](https://github.com/microsoft/PowerPlatformConnectors#certified-connectors) to create a Pull Request.

**Comment by seanspeaks** [comment]
@sriyen-msft I had followed those instructions, but I suppose I had been using an old fork and so when I merged upstream/master, it grabbed those changes as well.

I can create a new fork, if that's better? or remove those commits... I'll try the removing route first.

**Comment by seanspeaks** [comment]
Hmm @sriyen-msft I must be missing something? My fork is fully synced with `master` here, and my commit/PR is what I get when I make changes just to my target files (specifically, the `/certified-connectors/Huddle US` Directory)

**Comment by seanspeaks** [comment]
That's just an FYI. I'll revert those offending commits and I'll re-push

**Comment by seanspeaks** [comment]
Alright, @sriyen-msft All set I believe

**Comment by seanspeaks** [comment]
@sriyen-msft I saw that an automated check failed to pass (`info.title` was too long). Just FYI this did not surface in `paconn validate`, worth putting in there 👍

File: certified-connectors/Huddle US/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,896 @@
...
+          "x-ms-summary": "Title"
+        }
+      },
+      "type": "object"
+    }
+  },
+  "host": "msflow.huddle.com",
+  "info": {
+    "description": "**US GOVERNMENT & HEALTHCARE VERSION** (Standard version is also available.) Huddle is a secure document collaboration and client portal trusted by Enterprise and Government alike.",
+    "title": "Huddle for US Gov & Healthcare",
```
Comment (dougelliott) [inline certified-connectors/Huddle US/apiDefinition.swagger.json:309]: Given the 30 character limit for titles, perhaps 'Huddle (US Gov & Healthcare)'? Or similar?

**Review by dougelliott — CHANGES_REQUESTED** [review]

File: certified-connectors/Huddle US/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,896 @@
...
+          "type": "string",
+          "x-ms-summary": "Title"
+        }
+      },
+      "type": "object"
+    }
+  },
+  "host": "msflow.huddle.com",
+  "info": {
+    "description": "**US GOVERNMENT & HEALTHCARE VERSION** (Standard version is also available.) Huddle is a secure document collaboration and client portal trusted by Enterprise and Government alike.",
```
Comment (dougelliott) [inline certified-connectors/Huddle US/apiDefinition.swagger.json:308]: Could the asterisks please be removed? The rest of the description is good.

**Review by dougelliott — CHANGES_REQUESTED** [review]

**Comment by dougelliott** [comment]
> @sriyen-msft I saw that an automated check failed to pass (`info.title` was too long). Just FYI this did not surface in `paconn validate`, worth putting in there 👍

Thanks for letting us know Sean. @mamurshe as FYI on that.

File: certified-connectors/Huddle US/apiDefinition.swagger.json (hunk 1 above)
Comment (seanspeaks) [reply certified-connectors/Huddle US/apiDefinition.swagger.json:309]: Supposedly `Huddle for US Gov & Healthcare` is 30 characters... I didn't double check the automated validation, but my character count tool suggests that.

Are you seeing differently?

**Comment by seanspeaks** [comment]
Update made @dougelliott

File: certified-connectors/Huddle US/apiDefinition.swagger.json (hunk 1 above)
Comment (dougelliott) [reply certified-connectors/Huddle US/apiDefinition.swagger.json:309]: @seanspeaks You're absolutely right, it's 30 char. Apologies - a space char made its way into my offline check right at the end, bringing it to 31. Disregard the above, we're good to go on the title.

**Review by dougelliott — APPROVED** [review]

**Comment by seanspeaks** [comment]
Hey @dougelliott I'm confused by the checks here and your approval... looks like the build failed? or failed validation? but the errors are for things that actually exist? (info.contact and ms-metadata)

Thoughts?

**Comment by mamurshe** [comment]
@seanspeaks The validation failed because the PR contains other connectors in it. Can you please clean up your branch and resubmit?
A quick way to cleanup is to save your connector files locally, delete your fork, recreate a new fork, add your files in the fork and resubmit. Please note that the history for the fork will be lost with this method.
Another way is to use git rebase, which is a bit advanced, but recommended if you want to keep your fork history. Please look into online documentation if you want to learn more about git rebase.

**Review by mamurshe — CHANGES_REQUESTED** [review]
Please remove unrelated commit from the history.

**Pull request closed without merging**